packages/components/src/MultiSelect/MultiSelect.tsx

```diff
@@ -16,6 +17,10 @@ export type MultiSelectProps = {
   label: string
   items: MultiSelectOptionsProps["options"]
   selectedValues: Set<MultiSelectOption["value"]>
+  /**
+   * A description that provides context for the field
+   */
+  description?: ReactNode
```

Comment:
Could we use the type from `FieldMessage` itself? Just to make sure it's always passing the correct type